cscs-checks/prgenv/helloworld.py

```diff
@@ -121,6 +122,7 @@ class HelloWorldTestSerial(HelloWorldBaseTest):
     def __init__(self, lang, linkage):
         super().__init__('serial', lang, linkage)
         self.valid_systems += ['kesch:pn', 'arolla:pn', 'tsa:pn']
+        self.sourcesdir += os.path.join('/serial', lang)
```

Comment:
I don't see the reason of having directories per language. I think it's too much. We only have a single file per language.

Reply:
This comment applies everywhere.

Reply:
done